src/main/resources/assets/music_control/lang/en_us.json

```diff
@@ -13,9 +13,9 @@
 
     "text.autoconfig.music_control.title": "Music Control Configuration",
 
-    "text.autoconfig.music_control.option.timer": "Delay between musics",
+    "text.autoconfig.music_control.option.timer": "Maximum delay between musics[s]",
```

Comment:
[s] stands for seconds I believe, it took me some time to understand. You may keep it but please insert a space before for more legibility